tests/istio/authpolicy_controller_authorizationpolicy_test.go

```diff
@@ -365,7 +375,6 @@ var _ = Describe("AuthPolicy controller managing authorization policy", Ordered,
 				condition := meta.FindStatusCondition(existingPolicy.Status.Conditions, string(gatewayapiv1alpha2.PolicyConditionAccepted))
 				return condition != nil && condition.Reason == string(kuadrant.PolicyReasonUnknown) && strings.Contains(condition.Message, "cannot match any route rules, check for invalid route selectors in the policy")
 			}).WithContext(ctx).Should(BeTrue())
-
```

Comment:
nitpick: I would prefer to keep this blank line instead of removing it to keep the consistency across the other test cases

Reply:
@KevFan  added back in blank line for consistency across the other test cases.